jupyterlab/labapp.py

```diff
@@ -925,11 +925,11 @@ def initialize(self, argv=None):
             except ImportError:
                 self.log.critical(
                     """\
-Juptyer Lab cannot start, because `jupyter_collaboration` was configured but cannot be `import`ed.
+Jupyter Lab cannot start, because `jupyter_collaboration` was configured but cannot be `import`ed.
 
 To fix this, either:
 
-1) install the extension `jupyter_collaboration`; for example: `python -m pip install jupyter_collaboration`
+1) install the extension `jupyter-collaboration`; for example: `python -m pip install jupyter-collaboration`
```

Comment:
:+1: indeed, this is preferred. `pip` does not care, but conda does not normalize, and the feedstock uses `-` over `_` (https://github.com/conda-forge/jupyter-collaboration-feedstock)

Reply:
yes sorry! I copy-pasted into conda and had it fail which is why I opened this PR